pegasus/sites.v3/code.org/views/petition_expand.haml

```diff
@@ -82,9 +82,6 @@
 
   function Petition()
   {
-    // placeholder text for non-HTML5 browsers
-    $('input[type=text], textarea').placeholder();
-    $('input[type=email], textarea').placeholder();
```

Comment:
- the jquery.placeholder polyfill is [no longer needed](http://caniuse.com/#feat=input-placeholder) for modern browsers.
- `petition = new Petition();` and `petition.signPetition` can't go away, so I'm pretty sure the empty function definition needs to remain in order for the JS prototype-class to work (I could be wrong about this though)